extensions/dashlane-vault/src/lib/dcli.ts

```diff
@@ -2,21 +2,34 @@ import { getPreferenceValues } from "@raycast/api";
 import { existsSync } from "fs";
 import { safeParse } from "valibot";
 
-import { execFilePromis } from "@/helper/exec";
+import { getMasterPassword } from "@/helper/master-password";
 import { VaultCredential, VaultCredentialSchema, VaultNote, VaultNoteSchema } from "@/types/dcli";
+import { exec } from "child_process";
+import { execa } from "execa";
 
 const preferences = getPreferenceValues<Preferences>();
 
 const CLI_PATH =
   preferences.cliPath ?? ["/usr/local/bin/dcli", "/opt/homebrew/bin/dcli"].find((path) => existsSync(path));
 
 async function dcli(...args: string[]) {
-  if (CLI_PATH) {
-    const { stdout } = await execFilePromis(CLI_PATH, args, { maxBuffer: 4096 * 1024 });
-    return stdout;
+  if (!CLI_PATH) {
+    throw new Error("Dashlane CLI is not found!");
   }
+  const masterPassword = preferences.useTouchID ? await getMasterPassword() : null;
 
-  throw Error("Dashlane CLI is not found!");
+  const { stdout, stderr } = await execa(CLI_PATH, args, {
```

Comment:
This is not working on my machine. The promise never resolves. I updated the Dashlane CLI and set my master password. The Search Passwords command just shows me the loading state.

Reply:
Apparently my upstream commit for dashlane cli did not work, will look into this.